source/guides/getting-started/writing-your-first-test.md

```diff
@@ -437,6 +437,10 @@ Now Cypress provides us a UI (similar to debugger) to step forward through each
 
 {% video local /img/snippets/first-test-debugging-30fps.mp4 %}
 
+## Cypress fiddle
```

Comment:
I'm not sure I would put this in the getting started guide. I would prob put this in debugging. There's a bit of setup and understanding to get this going that I don't think people will have when writing their very first test.